src/containers/landing/components/section/FirstSection.tsx

```diff
@@ -6,34 +6,40 @@ import Card from "@/components/common/Card";
 import Button from "@/components/common/Button";
 import Spacing from "@/components/common/Spacing";
 import { Title } from "@/components/common/Title";
+import useWindowWidth from "@/hooks/useWindowWidth";
+
 const FirstSection = () => {
   const { reviewData } = FIRST_CONTENTS;
+  const width = useWindowWidth();
   const instaLink =
     "https://www.instagram.com/minery_app/?igshid=YmMyMTA2M2Y%3D";
+
+  const getButtonSize = () => {
+    if (width < 1199) return { width: 190, height: 48 };
+    return { width: 196, height: 60 };
+  }
+  
   return (
     <SectionWrapper>
       <DescWrapper>
-        <StyledTitle text="mainText" direction="row">
-          <p>와인으로 쓰는</p>
-          <p>그 날의 일기</p>
+        <StyledTitle text="mainText">
+          {width > 767 ? <div>와인으로 쓰는 그 날의 일기</div> : <><div>와인으로 쓰는</div><div>그 날의 일기</div></>}
```

Comment:
React Element를 조건부로 처리하는 건 좋지 않아 보여요.
특히 Next.js는 SSG나 SSR로 서버에서 미리 내려주는데 이 때, width를 서버에서는 알 수 없기 때문에 디자인이 깨질 우려가 있어요.
차라리 \n을 인식할 수 있는 pre 태그로 미리 감싸두고 표현하는 내용만 다르게 하는게 좋다고 생각해요.

```suggestion
        <pre>
          {width > 767 ? "와인으로 쓰는 그 날의 일기" : `와인으로 쓰는\n그 날의 일기`}
        </pre>
```